pkg/events/parse_args.go

```diff
@@ -274,24 +275,54 @@ func ParseArgsFDs(event *trace.Event, origTimestamp uint64, fdArgPathMap *bpf.BP
 	return nil
 }
 
-func GetArg(event *trace.Event, argName string) *trace.Argument {
-	for i := range event.Args {
-		if event.Args[i].Name == argName {
-			return &event.Args[i]
+func GetArg(args []trace.Argument, argName string) *trace.Argument {
+	for i := range args {
+		if args[i].Name == argName {
+			return &args[i]
 		}
 	}
+
 	return nil
 }
 
 func SetArgValue(event *trace.Event, argName string, value any) error {
-	arg := GetArg(event, argName)
+	arg := GetArg(event.Args, argName)
 	if arg == nil {
 		return fmt.Errorf("event %s has no argument named %s", event.EventName, argName)
 	}
 	arg.Value = value
 	return nil
 }
 
+// NormalizeTimeArgs normalizes the time arguments of an event, converting them to
+// nanoseconds since the epoch.
+func NormalizeTimeArgs(eventID ID, args []trace.Argument, timeArgNames []string) error {
```

Comment:
nit: I don't think we should pass the eventID just for the error message in the callee, the caller should handle that.